src/api.mjs

```diff
@@ -107,6 +107,32 @@ async function route(message) {
       return { ...message, error: error.toString() };
     }
     return { ...message, results: data };
+  } else if (type === "arweave") {
+    const { uri, timeout: timeoutFromMsg } = message.options;
+
+    const ARWEAVE = "https://arweave.net";
```

Comment:
it'd be better if we had a `gateway` property similar to how the IPFS message works. Is there a reason we preferred to hard code the gateway here?